Engine/Setup/BacktestingSetupHandler.cs

```diff
@@ -101,12 +102,15 @@ public int MaxOrders
             }
         }
 
+        public IExceptionParser ExceptionParser { get; }
```

Comment:
Instead of parsing the exceptions from the set up handler, perhaps it's best to convert the `List<string> Errors>` into a `List<Exception` -- then the engine can do it's thing on the exceptions. This would remove the exception parsing dependency from the set up handlers and keep it isolated to the engine code.